config/routes.rb

```diff
@@ -24,10 +24,10 @@
     end
 
     resources :funds, only: [], concerns: [:activity, :transactionable] do
-      resources :programmes, only: [:new, :create, :show]
+      resources :programmes, except: [:destroy]
     end
-    # TODO: Extend with more hierarchies using this format
-    # resources :programmes, only: [], concerns: [:activity, :transactionable]
+
+    resources :programmes, only: [], concerns: [:activity, :transactionable]
```

Comment:
Should we keep the comment in place until we have all four levels implemented..?